scripts/DR_distributions_cross_run_analysis.py

```diff
@@ -0,0 +1,228 @@
+USAGE = """
+
+Summarize deed-restricted units by source, year-built, and locations.
+
+"""
+
+import glob, os, time
+import pandas as pd
+import numpy as np
+
+
+# set up file directory and run_id dictionary
+# when we have new runs, copy the 'building_data_2050.csv' output to this directory,
+# and add the new run_id and BAUS version to runid dictionry 
+
+run_dir = 'M:\\Data\\Urban\\BAUS\\PBA50\\Final_Blueprint\\run_analysis\\DR analysis'
+os.chdir(run_dir)
+
+# run_id dictionary
+runid = {'run325': 'v2.2.1',
+         'run241': 'v2.3',
+         'run242': 'v2.3.1',
+         'run152': 'v2.4',
+         'run327': 'v2.5',
+         'run243': 'v2.6',
+         'run158': 'v2.7',
+         'run335': 'v2.8',
+         #'run160': 'v2.9',
+         #'run253': 'v2.10',
+         #'run161': 'v2.11',
+         #'run340': 'v2.12',
+         'run254': 'v2.13',
+         'run171': 'v2.14',
+         'run347': 'v2.15',
+         'run261': 'v2.16'}
+
```

Comment:
Consider making this mapping a config file that is useful for other scripts/processes
It could richer information as well (which version was used for milestone documents, etc.)